### ISSUE TITLE
Cannot open TSM

### ISSUE THREAD

**Issue by Casteele72** [sienori/Tab-Session-Manager#723]
I can no longer open TSM. Either it gives the "indexedDB" error (rarely, maybe one time out of twenty), which I have followed the instructions to fix--to no avail; OR, most often, it gives the following error:
"Error
Multiple popups can not be opened at the same time."

I've disabled and enabled TSM. I've done the same with all extensions. I've started Firefox in safe mode. I've made sure to close every window and tab in Firefox, shut it down, and restart it with nothing but a single window with a single blank tab. Yet it still thinks there is another popup open somewhere.

OS:
Linux 5.4.0-0.bpo.4-amd64 #1 SMP Debian 5.4.19-1~bpo10+1 (2020-03-09) x86_64 GNU/Linux

Firefox Version:
76.8.0esr (64-bit)

TSM Version
5.2.0

Side note: I've also had many problems with restoring sessions under Tree Style Tabs, as well. Sometimes the tree does not restore properly, or out of order, or even times the tabs are all over the place or not showing up at all--even though they continue to exists if I CTRL-PgUp/PgDn to switch through them.

**Comment by sienori** [comment]
The latest version of TSM is 6.4.1. Can you update it?

**Comment by Casteele72** [comment]
Updated. It seems to be working now--Except all my previously saved sessions are missing. It's not a major issue to me, since I exported and backed up all my sessions except for the last few days when the problem started. As a suggestion, add a feature (to the add-on settings from the add-on manager, since that doesn't seem to get clobbered) to "Export/import a full backup of all sessions" or some such.
